packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -1062,6 +1063,8 @@ describe('IP blocking', function () {
       }
     }))
 
+    disableLfi()
```

Comment:
why? I think that it worth to add a comment in the code to understand this unrelated change.